### ISSUE TITLE
Color Area suggested markup has duplicated input labels

### ISSUE THREAD

**Issue by GarthDB** [adobe/spectrum-css#1262]
## Description

Currently

```html
<input type="range" class="spectrum-ColorArea-slider" name="x" aria-label="saturation and value" min="0" max="1" step="0.01">
<input type="range" class="spectrum-ColorArea-slider" name="y" aria-label="saturation and value" min="0" max="1" step="0.01">
```

The 2 different inputs should have 2 different aria-labels.

## Link to documentation

https://opensource.adobe.com/spectrum-css/colorarea.html

synced to jira: https://jira.corp.adobe.com/browse/SDS-10302

**Comment by pfulton** [comment]
These values were revised and corrected during the migration to `@adobe/spectrum-tokens`
